### ISSUE TITLE
[Draft] aria-* values treated differently if they are capitalized

### ISSUE THREAD

**Issue by daniel-montalvo** [auto-wcag/auto-wcag#2276]
## ACT Rules

- [ARIA required ID references exist]()

### Test case 1 ...

(( Add test case here ))

```html
code snippet here
```

[Open this test case]()

(( Add a link to a code pen or other hosted test case. Keep in mind this test case needs to be accessible and persistent. ))

#### Preconditions

Before running the test, check your settings. In many cases default settings aren't strictly required and will not impact the results. Use common sense here. If non-default settings are necessary for you, record the settings used in the reported results.

- Ensure browsers are in the default settings, turn off unnecessary extensions
- Ensure ATs are in the default settings
- Ensure operating system are in default settings
- Ensure software uses the correct version (usually the latest version)

#### Test instructions

(( Provide instructions on how to test the test case, this should include: ))

- Which technologies to use
- Any setting changes necessary to run the test
- Any variation in how to operate the different assistive technologies
- Step by step instructions on how to check the test case
- What the expected results are

### External links

(( List any external links to bugs in browsers & AT, WPT, or spec issues. Be sure to open issues in the appropriate repositories. ))

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case X

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Assistive technology: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```

**Comment by shunguoy** [comment]
Per [ARIA in HTML](https://w3c.github.io/html-aria/#case-sensitivity):
  

> Authors SHOULD use [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase) for all role token values and any state or property attributes (aria-*) whose values are [defined as tokens](https://www.w3.org/TR/wai-aria-1.2/#propcharacteristic_value).
